assets/scss/components/_service-info.scss

```diff
@@ -195,3 +195,48 @@ Styleguide Service Info.Logged In
 .last-login__more-details {
   margin: em(5) 0 0 0;
 }
+
+/*
+Floated items
+
+Service Info with `.service-info__floated-items` allows for content to be floated on the left and right sides with adequate margins. On mobile breakpoints content is displayed block
+
+Markup:
+<div class="service-info ">
+  <div class="service-info__floated-items">
+    <div class="service-info__left-item">
+      <span>Left Side</span>
+    </div>
+    <div class="service-info__right-item">
+      <span>Right Side</span>
+    </div>
+  </div>
+</div>
+
+Styleguide Service Info.Left and Right Items
+*/
+.service-info__floated-items {
```

Comment:
For now, there's a growing [list of selectors](https://github.com/hmrc/assets-frontend/blob/master/assets/scss/components/_service-info.scss#L48-L50) that are already doing this job. can you add to that or use one of them instead of adding a wrapper here?

We should probably be adding padding to `.service-info` instead when we refactor this stuff (maybe a github issue for now?)